src/components/AddTask.tsx

```diff
@@ -0,0 +1,79 @@
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { getFormattedDate } from "../utils/dateFunctions";
+import circledPlusIcon from '../assets/icons/circledPlusIcon.png'
+import { AddTaskProp } from "../types/types";
+
+export function AddTask(props: AddTaskProp) {
+
+    return (
+        <>
+            <div className={props.isModalOpen ? "add-task-container" : "add-task-container zIndex"} data-testid="container">
+                <div className="description-project-container">
+                    <input
+                        data-testid="task-name"
+                        type="text"
+                        placeholder="What are you working on?"
+                        className="input-box"
+                        onChange={props.onNameChange} 
+                        value={props.taskDescription}
+                        onKeyDown={props.onEnter}
+                    ></input>
+                    <button onClick={props.onToggle} className={!props.selectedProject.value ? "project-text-color project-text" : 'project-text'}>
+                        {!props.selectedProject.value && <img src={circledPlusIcon} alt="Circled Plus Icon" style={{ width: '20px', height: '20px', marginRight: '5px'}}/>}
+                        {`${props?.selectedProject?.label}${(props?.selectedClient?.label && props.selectedProject?.value) && ' - '}${props.selectedProject?.value ? props?.selectedClient?.label : ''}`}
```

Comment:
What is this for? Anyway to extract and make this more readable?

Reply:
That logic is for showing selected Project and client names Anna. Extracted the logic outside of the component.

https://github.com/PreethiKalyani99/Clockify-Typescript/commit/e27c71d47043d1837192129035b4cc01ae8b42e5